src/test/java/hudson/plugins/performance/workflow/WorkflowActionsFactoryTest.java

```diff
@@ -39,6 +41,33 @@ public void evaluate() throws Throwable {
                 r = p.scheduleBuild2(1).waitForStart();
                 story.j.assertBuildStatus(Result.SUCCESS, story.j.waitForCompletion(r));
                 Collection<? extends Action> actions = factory.createFor(p);
+                assertEquals(0, actions.size());
+            }
+        });
+    }
+
+    @Test
+    public void testFlowWithProjectAction() throws Exception {
```

Comment:
mhhhhn thats not true: https://github.com/rpionke/performance-plugin/blob/f055afc0602556f2cece6e1656f0a85cea8ee64e/src/test/java/hudson/plugins/performance/workflow/WorkflowActionsFactoryTest.java#L71-L72